src/mds/MDSRank.h

```diff
@@ -151,6 +151,28 @@ class Finisher;
 class ScrubStack;
 class C_ExecAndReply;
 
+struct MDSMetaRequest {
+private:
+  int op;
+  CDentry *stray_dentry;
```

Comment:
```suggestion
  CDentry *stray_dentry;
```

I'd prefer to keep the name generic: `dentry`.

Reply:
Sure. I will fix it.